hudi-spark-datasource/hudi-spark/src/test/java/org/apache/hudi/functional/TestOrcBootstrap.java

```diff
@@ -169,11 +170,13 @@ public Schema generateNewDataSetAndReturnSchema(long timestamp, int numRecords,
     return AvroOrcUtils.createAvroSchemaWithDefaultValue(orcSchema, "test_orc_record", null, true);
   }
 
+  @Disabled
   @Test
   public void testMetadataBootstrapNonpartitionedCOW() throws Exception {
     testBootstrapCommon(false, false, EffectiveMode.METADATA_BOOTSTRAP_MODE);
   }
```

Comment:
you can disable for the whole class instead of annotating each method, and add a note in the annotation like `@Disabled("<describe the reason>")`